docs/News.md

```diff
@@ -1,4 +1,12 @@
 ### News
+
+#### February 7, 2018
+ *   **Oncotree format expanded to support deeper tree nodes**
+        * To support upcoming expansion of the oncotree, the 5 named levels of the oncotree {*Primary*, *Secondary*, *Tertiary*, *Quaternary*, *Quinternary*} have been dropped in favor of level numbers {1, 2, 3, 4, 5, ...}. Web API functions have been adjusted accordingly, and an API function which outputs a table format of the oncotree has been adjusted to output 7 levels of depth.
+        * Additional improvements to the website:
+                * tab-style navigation
```

Comment:
When you view this "* tab-style navigation ... * a **News** page" is all on one line.

Reply:
I see .. I'll fix that